src/content/pages/partnerships.mdx

```diff
@@ -0,0 +1,116 @@
+---
+seo:
+  title: Partner with Astro
+  description: Want to partner with Astro?
+---
+
+import StatsFeature from '~/components/StatsFeature.astro'
+import Note from '~/components/PartnershipNote.astro'
+
+![](../../assets/partnerships-page-1.webp)
+
+**Let's improve the web, together!**
+
+## ❤ Sponsorships and Official Partners
+
+The easiest way to partner with Astro is to become a sponsor! Your financial contributions support the open-source project's ongoing maintenance and developent.
+
+We recognize some levels of sponsorship with a logo and/or link on Astro sites such as our homepage, GitHub README, and docs website.
+
+<Note color="purple">
+  💸 **To sponsor:** Visit [Open Collective](https://opencollective.com/astrodotbuild) or [GitHub Sponsors](https://github.com/sponsors/withastro) and choose from a list of contribution tiers. Thank you for supporting Astro!
+</Note>
+
+If you don't see your desired level of sponsorship listed, please contact us at `partner@astro.build`!
+
+We are always happy to work directly with larger sponsors ($2000+ per month) to create a custom sponsorship package. There are also limited options for being an exclusive "official" partner for a particular category (e.g. Official CMS Partner).
```

Comment:
exclusive sponsor on github (and soon opencollective?) is 10k, so maybe the amount needs to be updated here?

I mean I find it a bit confusing:
- It mentions 2k+ when tiers are (formerly) 1k or 2k
- It also mention exclusive partnership as another option, which is...2k

So yeah probably worth clarifying some stuff

Reply:
I don't think it matters too much that these numbers are different. It says, we're happy to work with larger sponsors to create something custom. $2K is defining what we think of as "larger"? So maybe a good number to mention, because it's something *between* the tiers of 1K and 10K? (the new tiers, updated!)

Could be a good way to get "larger than the currently largest tier that exists" (since official partnerships are going to be limited, and someone with money might not fit into an available category)? It's like letting someone know that "$2K could be a thing, too! What do you want?"